my-school/src/Components/Dashboards/AdminDash.js

```diff
@@ -19,6 +20,10 @@ const AdminDash = ({ user, isLoading }) => {
     ReactGA.pageview("/dashboard")
   }, [])
 
+  useEffect(() => {
+    getAllActivities()
+  }, [])
```

Comment:
Does this need to get all? If this is for the "recent activity" preview we may do well to just query the most recent for each student